### PR TITLE
[Toasts] PortalContent as message

### PR DESCRIPTION
**Pull request by guillerot**
## Description

Use PortalDirective with LuToastsComponent to allow use of TemplateRef

-----

https://github.com/LuccaSA/lucca-front/assets/2039493/6adbcc3f-ddae-473f-b275-210f9434c6f8

### CODE REVIEW

File: packages/ng/toast/toasts.component.html [hunk 1]
```diff
@@ -3,7 +3,11 @@
 		<span *ngIf="toast.type" class="toast-item-icon lucca-icon" aria-hidden="true" [ngClass]="iconClassByToastType[toast.type]"></span>
 		<div class="toast-item-content">
 			<div *ngIf="toast.title"><strong>{{ toast.title }}</strong></div>
+			@if (isStringPortalContent(toast.message)) {
```
Comment (GuillaumeNury) [inline packages/ng/toast/toasts.component.html:6]: You had to do this for `innerHTML` support? (`*luPortal` accepts string but not HTML I guess)

**Review by GuillaumeNury — APPROVED** [review]
Thanks!

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2853/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/18115/ui-diff)

File: packages/ng/toast/toasts.component.html (hunk 1 above)
Comment (guillerot) [reply packages/ng/toast/toasts.component.html:6]: Yes.

**Review by jeremie-lucca — APPROVED** [review]

**Pull request merged**